spec/system/sidebar_topic_button_spec.rb

```diff
@@ -33,13 +33,13 @@
     expect(page).to have_css(".sidebar-new-topic-button__wrapper .topic-drafts-menu-trigger")
   end
 
-  it "disables button when visiting read-only category" do
+  it "enables button when visiting read-only category" do
```

Comment:
Hmm... I guess we want to show a non-disabled button in a category where the user can't post? Is that what we are doing in core too now?